errored.go

```diff
@@ -152,11 +152,13 @@ func (e *Error) Error() string {
 	}
 
 	if e.trace || AlwaysTrace {
-		ret := desc + "\n"
-
-		for _, stack := range e.stack {
-			for _, line := range stack {
-				ret += fmt.Sprintf("\t%s [%s %d]\n", line.fun, line.file, line.line)
+		var ret string
+		for i := 0; i < len(e.errors); i++ {
+			ret += fmt.Sprintf("%s\n", e.errors[i].Error())
+			if len(e.stack) > i {
```

Comment:
`Combine()` accepts error but it converts to *Error via encapsulation, so all is good :) We even get a stack trace at Combine time.